<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>521</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
    <property name="horizontalSpacing">
     <number>8</number>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="SelectedMesh_QLineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="PopulateMesh_QPushButton">
      <property name="text">
       <string>Get Selection</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="ExportMesh_QLabel">
      <property name="text">
       <string>Exportable Mesh:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="BuildExportRig_QPushButton">
      <property name="text">
       <string>Build Export Rig</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ExportAnim_QLabel">
      <property name="text">
       <string>Exportable Anim:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="SelectedJoint_QLineEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="PopulateJoint_QPushButton">
      <property name="text">
       <string>Get Selection</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="BakeAnim_QPushButton">
      <property name="text">
       <string>Bake onto Export Joints</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Version_QLabel">
      <property name="text">
       <string>Febex 1.0.0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Authorship_QLabel">
      <property name="text">
       <string>Matt Riche 2023 (MIT-0)  </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
